runner.py

```diff
@@ -32,6 +34,16 @@ def isUpToDate() -> bool:
     return res
 
 
+def db_init():
+    dbConn = database.create_connection()
+    database.db_set_up(dbConn)
+    ppl = tamer.get_person_names()

```

Comment:
```suggestion
    ppl = tamer.get_ppl_names()

```

currently you're calling the deprecated function, which makes the program crash on start

Reply:
Thanks for spotting!